src/phpDocumentor/Descriptor/Builder/PhpParser/FunctionAssembler.php

```diff
@@ -40,11 +40,11 @@ public function __construct(ArgumentAssembler $argumentAssembler)
      *
      * @param Function_ $data
      *
-     * @return FunctionDescriptor
+     * @return Function_
      */
     public function create($data)
     {
-        $functionDescriptor = new FunctionDescriptor();
+        $functionDescriptor = new Function_();
```

Comment:
Just reviewed the Function_ element and this Assembler should be updated to match the new constructor